reference/security-compliance/faq.md

```diff
@@ -0,0 +1,114 @@
+---
+title: FAQ
+description: Frequently asked questions regarding Security & Compliance
+---
+
+Detailed below are the answers to some frequently asked questions about PlaceOS, data storage and security.
+
+## Data
+
+**Q: How does PlaceOS manage data, protecting customer data and governs customer data?**
+
+A: Most data is not stored internally within the system. 
```

Comment:
Some of these answers are vague and avoidant, I say we merge this in for now (because something is better than nothing) and then keep this branch open and create another PR in the future when we can DETAIL the exact fields which are stored and where.

So the goal would create a document that's much more detailed than [this older ACA one](https://docs.google.com/document/d/1hCTJ9fZERB8n1i7IKyLkDP43zH8K92HS9JANglwN84g/edit#heading=h.d8eyolxc71b8)